src/chapters/classes/encapsulation.rst

```diff
@@ -0,0 +1,238 @@
+   a public getter and a package-private setter, but not as a field
+   without getters and setters, which could only be public to everyone
+   or package-private to everyone.
+
+As an example of reason 2, let’s take a short detour to look at a
+``Temperature`` class. A valid temperature can only be so low (“absolute
+zero”), so we wouldn’t want to allow somebody to set an invalid value.
+In ``setFahrenheit`` we throw an exception if an invalid value is
+provided (we’ll cover exceptions in detail later, but for now note that
+they are ways of signaling errors).
+
+.. sourcecode:: java
+   :linenos:
+
+   public class Temperature {
+
+       private double fahrenheit;
+
+       public double getFahrenheit() {
+           return fahrenheit;
+       }
+
+       public void setFahrenheit(double aFahrenheit) {
+
+           double absoluteZeroFahrenheit = -459.67;
+
+           if (aFahrenheit < absoluteZeroFahrenheit) {
+               throw new IllegalArgumentException("Value is below absolute zero");
+           }
+
+           fahrenheit = aFahrenheit;
+       }
+   }
+
+.. note::
+
+   When writing getters and setters, the convention for a field named
+   ``field`` is to name them ``getField`` and ``setField``. This is more
+   than just a convention, as some libraries you use will *expect* names to
+   be of this format, and won’t work as desired if you don’t follow the
+   convention.
+
+   Additionally, it’s a standard convention to use ``is`` instead of
+   ``get`` for boolean fields. So a boolean field ``oldEnoughToVote`` would
+   have the “getter” method ``isOldEnoughToVote``. The setter should still
+   be named ``setOldEnoughToVote``.
+
+Properties
+^^^^^^^^^^
+
+A **property** in Java is a characteristic that users can set. Our
+``Student`` class had properties ``name``, ``studentId``,
+``numberOfCredits``, and ``gpa``, while our ``Temperature`` class had
+only one property, ``fahrenheit``.
+
+Most often, properties will be fields that have public setters, though
+they need not have a corresponding field. Let’s look at an example of a
+property that doesn’t directly correspond to a field. If we wanted to
+add a ``celsius`` property to the ``Temperature`` class above, we might
+do it as follows:
+
+.. sourcecode:: java
+   :linenos:
+
+   public double getCelsius() {
+       return (fahrenheit - 32) * 5.0 / 9.0;
+   }
+
+   public void setCelsius(double celsius) {
+       double fahrenheit = celsius * 9.0 / 5.0 + 32;
+       setFahrenheit(fahrenheit);
+   }
+
+Since there’s a link between Fahrenheit and celsius, we want to make
+sure that when one is updated, so is the other. In this case, we only
+store one field value (``fahrenheit``) and make the appropriate
+calculation when getting or setting the celsius property.
+
+.. note::
+
+   There are slight variations among Java developers when it comes to
+   colloquial usage of the term “property”. People will sometimes define
```

Comment:
instead of using quotation marks around property, id change it to be italicized per our style guide

Reply:
Done! Thanks!